### PR TITLE
Move stars counters into button

### PR DESCRIPTION
**Pull request by ivarpruijn**
Moves the stars counter into the button, instead of next to it. Now looks like this:

image

### CODE REVIEW

**Comment by jansenbob** [comment]
All LGTM, but i'm not a big fan of the blue hover color:

image

Maybe we can just leave the button background color as is, as the drop shadow already indicates that you're hovering the button.

**Review by jansenbob — CHANGES_REQUESTED** [review]
change blue background on hover to FAFAFA

File: src/pages/index.tsx [hunk 1]
```diff
@@ -199,7 +199,7 @@ export default function Home() {
                   <p>
                     Objectiv&nbsp;
                     <TrackedLink
-                      to={useBaseUrl("/docs/modeling/example-notebooks/", {absolute: true})}
+                      to={useBaseUrl("/docs/modeling/open-model-hub/", {absolute: true})}
```
Comment (ivarpruijn) [inline src/pages/index.tsx:202]: @jansenbob @vincenthoogsteder Updated the link on the homepage to take you directly to the open model hub docs. Thought that made more sense than the example notebooks or the introduction page of the modeling section. WDYT?

Comment (vincenthoogsteder) [reply src/pages/index.tsx:202]: agreed! 

shall we also do that with the GH readme then? That now points to modeling intro, but agree this is better

**Review by jansenbob — APPROVED** [review]
LGTM!

File: src/pages/index.tsx (hunk 1 above)
Comment (ivarpruijn) [reply src/pages/index.tsx:202]: let's!

**Pull request merged**